mybatis-plus-boot-starter/src/main/resources/META-INF/spring.factories

```diff
@@ -1,7 +1,3 @@
 # Auto Configure
 org.springframework.boot.env.EnvironmentPostProcessor=\
   com.baomidou.mybatisplus.autoconfigure.SafetyEncryptProcessor
-org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
-1, I don't agree with this change, it breaks compatibility with Springboot OSS < 2.7.0, it should be kept.  Reference https://github.com/remkop/picocli/pull/1778#issuecomment-1242654922 .

Reply:
> 

> 就目前而言，我觉得也可以不删除，但是springboot3已经删除了这个装配的方式[#29699](https://github.com/spring-projects/spring-boot/issues/29699)

Reply:
- Yes, I meant to have the new file exist alongside the auto-config section of `spring.factories`.